tests/Kernel/ParagraphsTest.php

```diff
@@ -136,6 +137,31 @@ public function testAccordions() {
     $this->assertEquals(1, $actual);
   }
 
+  /**
+   * Test quote paragraph rendering.
+   */
+  public function testQuotes() {
+
+    $attribution = 'Quote author goes here';
+    $body = 'Quote body goes here';
```

Comment:
I would prefer to use a dataprovider in this case, I think it would be better to do that all the time to avoid mixing the logic of the test AND the test case data.
@ademarco what's your point of view on this ?

Reply:
@ademarco and @drupol I fine to create a new public method for the data provider, however, it seems me that it is useful only when the purpose is to test several cases. Let's discuss it...

Reply:
This is true as well.
I was higlighting this in a matter of code consistency.
I leave @ademarco decide on this.